pywatts/core/step.py

```diff
@@ -108,9 +108,10 @@ def load(cls, stored_step: Dict, inputs, targets, module, file_manager):
             callback.set_filemanager(file_manager)
             callbacks.append(callback)
 
-        step = cls(module, inputs, targets=targets, file_manager=file_manager,
-                   computation_mode=ComputationMode(stored_step["computation_mode"]), condition=condition,
+        step = cls(module, inputs, targets=targets, file_manager=file_manager, condition=condition,
                    train_if=train_if, callbacks=callbacks, batch_size=stored_step["batch_size"])
+        step.default_run_setting = RunSetting.load(stored_step["default_run_setting"])
```

Comment:
When testing the examples, the pipeline crashes due to KeyError 'default_run_setting'. I do not find the problem there but I think it is a bug.